Source/Xamarin/Prism.Forms/Navigation/Xaml/GoBackExtension.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Xamarin.Forms;
+
+namespace Prism.Navigation.Xaml
+{
+    [ContentProperty(nameof(GoBackType))]
+    public class GoBackExtension : Navigation
+    {
+        public bool Animated { get; set; } = true;
+        public GoBackType GoBackType { get; set; } = GoBackType.Default;
+        public bool? UseModalNavigation { get; set; } = null;
+
+        public override async void Execute(object parameter)
+        {
+            var parameters = parameter.ToNavigationParameters(Bindable);
+
+            IsNavigating = true;
+            RaiseCanExecuteChanged();
+
+            var navigationService = GetNavigationService(Page);
+
+            if (GoBackType == GoBackType.ToRoot)
```

Comment:
let's also do this in a virtual Handle method to make this more extensible for custom fault handling

Reply:
Done